### PR TITLE
Add BFP to omnibus mainnet

### PR DESCRIPTION
**Pull request by 0xMithrandir**
Adds a basic BFP configuration for mainnet
Tests are added for common operations like opening positions, liquidations, withdrawals, etc.
Pyth feed was updated to mainnet address as well as Chainlink.

### CODE REVIEW

File: tomls/omnibus-mainnet/bfp.toml [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+minKeeperFeeUsd = "<%= settings.min_keeper_fee_usd %>"
+maxKeeperFeeUsd = "<%= settings.max_keeper_fee_usd %>"
+keeperProfitMarginUsd = "<%= settings.keeper_profit_margin_usd %>"
+keeperProfitMarginPercent = "<%= settings.keeper_profit_margin_percent %>"
+keeperSettlementGasUnits = 1200000
+keeperCancellationGasUnits = 600000
+keeperFlagGasUnits = 1200000
+keeperLiquidateMarginGasUnits = 1200000
+keeperLiquidationGasUnits = 1200000
+keeperLiquidationEndorsed = "<%= settings.deployer %>"                         # TODO, this is currently deployer, change to a keeper
```
Comment (noisekit) [inline tomls/omnibus-mainnet/bfp.toml:134]: we need to resolve this todo

**Review by noisekit — COMMENTED** [review]
one todo and also need to increment version in omnibus-mainnet.toml

File: tomls/omnibus-mainnet/bfp.toml (hunk 1 above)
Comment (0xMithrandir) [reply tomls/omnibus-mainnet/bfp.toml:134]: Yes, this keeper address is whitelisted to exceed liquidation caps.

**Review by noisekit — APPROVED** [review]
LFG

**Pull request merged**